docs/observability/metrics.md

```diff
@@ -99,6 +99,9 @@ Virtual Machine last transition timestamp to error status. Type: Counter.
 ### kubevirt_vm_info
 Information about Virtual Machines. Type: Gauge.
 
+### kubevirt_vm_labels
+The metric exposes the VM labels as Prometheus labels. Configure allowed and ignored labels via the 'kubevirt-vm-labels-config' ConfigMap. Type: Gauge.
```

Comment:
How will they know where to find the 'kubevirt-vm-labels-config' ConfigMap example?

Reply:
I don't think so since it's the help section.. it will not be a link it will be text